### PR TITLE
LPS-101624, LPS-101625 Deprecate `Liferay.Util.isTablet` and `Liferay.Util.isPhone`

### PR DESCRIPTION
**Pull request by julien**
Originally discussed in [#206](https://github.com/wincent/liferay-portal/pull/206)

The goal of these changes is to mark `Liferay.Util.isTablet` and
`Liferay.Util.isPhone` as deprecated. We're also providing these
functions as "modules" so that they can easily be imported projects
that use ES6.

@wincent in case you're confused; you already reviewed this yesterday 
in [#213](https://github.com/wincent/liferay-portal/pull/213) and it was forwarded to [https://github.com/brianchandotcom/liferay-portal/pull/87789](https://github.com/brianchandotcom/liferay-portal/pull/87789), but based on [this](https://github.com/brianchandotcom/liferay-portal/pull/87789#issuecomment-616785319) 
comment, I updated the JSDoc `@deprecated` comments.

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js [hunk 1]
```diff
@@ -61,6 +64,11 @@ import toCharCode from './util/to_char_code.es';
 
 Liferay = window.Liferay || {};
 
+/**
+ * @deprecated As of Athanasius (7.3.x), replaced by `import BREAKPOINTS from 'frontend-js-web'`
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js:68]: This seems wrong. It should be:

```
* @deprecated As of Athanasius (7.3.x), replaced by `import {BREAKPOINTS} from 'frontend-js-web'`
```

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js [hunk 2]
```diff
@@ -109,6 +117,17 @@ Liferay.Util.getFormElement = getFormElement;
...
 Liferay.Util.groupBy = groupBy;
 Liferay.Util.isEqual = isEqual;
+
+/**
+ * @deprecated As of Athanasius (7.3.x), replaced by `import isPhone from 'frontend-js-web'`
+ */
+Liferay.Util.isPhone = isPhone;
+
+/**
+ * @deprecated As of Athanasius (7.3.x), replaced by `import isTablet from 'frontend-js-web'`
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js:127]: Same here:

```
 * @deprecated As of Athanasius (7.3.x), replaced by `import {isPhone} from 'frontend-js-web'`
```

and

```
 * @deprecated As of Athanasius (7.3.x), replaced by `import {isTablet} from 'frontend-js-web'`
```

**Comment by wincent** [comment]
Let's try this one again.

**Comment by wincent** [comment]
ci:forward

**Comment by wincent** [comment]
Error seems legit:

```
Module not found: Error: Can't resolve './util/is_tablet' in '/opt/dev/projects/github/liferay-portal/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay'
```

I'm going to pull this down and see if I can repro and fix seeing as it has been blocked in one way or another for a few days now.

**Comment by wincent** [comment]
@julien: see https://github.com/wincent/liferay-portal/pull/218/commits/98ec820b45830faedda4f19eca69df410be4ebd7

3 files didn't come across when you resent #213 here, and the horrible GitHub UI doesn't make it easy to see among all the crap:

CleanShot 2020-04-22 at 11 15 17@2x

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
Local build works:

CleanShot 2020-04-22 at 11 47 25@2x

Functions exist and work:

CleanShot 2020-04-22 at 12 05 39@2x

**Comment by wincent** [comment]
ci:forward

**Comment by wincent** [comment]
Ok, well that obviously didn't work. No updates from CI despite being "pending" for 24 hours, so going to try again.

**Comment by wincent** [comment]
ci:stop

**Comment by wincent** [comment]
ci:forward

**Pull request closed without merging**